scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -0,0 +1,65 @@
+[[scaling-performance-compute-resources]]
+= Optimizing Compute Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-overcomitting]]
+== Overcommitting
+
+You can use overcommit procedures so that resources such as CPU and memory are
+more accessible to the parts of your cluster that need them.
+
+Note that when you overcommit, you take the risk that another application may
```

Comment:
s/you take the risk/there is the risk 
or
s/you take the risk/you assume the risk